### PR TITLE
{bio}[GCC/11.3.0] EPA-ng v0.3.8, PLL-modules v20230628, genesis v0.29.0, ...

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
tcn445.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/485c4edab17015f8d84477d555cfda1f for a full test report.

File: easybuild/easyconfigs/e/EPA-ng/EPA-ng-0.3.8-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'EPA-ng'
+version = '0.3.8'
+
+homepage = 'https://github.com/pierrebarbera/epa-ng'
+description = """
+EPA-ng - Fast, parallel, highly accurate Maximum Likelihood Phylogenetic
```
Comment (casparvl) [inline easybuild/easyconfigs/e/EPA-ng/EPA-ng-0.3.8-gompi-2022a.eb:8]: Doesn't this create a new line to start with? E.g. https://github.com/easybuilders/easybuild-easyconfigs/pull/18420/files starts the description right after the `"""`

File: easybuild/easyconfigs/g/genesis/genesis-0.29.0-GCC-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.2.5'),
+    ('HTSlib', '1.15.1'),
+    ('libdeflate', '1.10'),
+]
+
+skipsteps = ['install']
+
+postinstallcmds = ['mkdir -p %%(installdir)s/lib/ && '
```
Comment (casparvl) [inline easybuild/easyconfigs/g/genesis/genesis-0.29.0-GCC-11.3.0.eb:32]: I was checking our Generic EasyBlocks. It's funny: we have something for software without `configure`, with `make`, and without `make install` (`makeCp`), as well as for software without `configure`, `make` and `make install` (`cmdCp`), but _not_ for software _with_ a `configure`, a `make`, but no `make install`.

We _should_ really have a `ConfigureMakeCp`... But don't worry, I won't force you to make one (but if you feel like it, I'd encourage it ;-))

**Review by casparvl — COMMENTED** [review]
EasyConfigs look good to me. Let's wait for the tests, if they succeed, this can be merged.

**Review by casparvl — CHANGES_REQUESTED** [review]
Oh, I forgot: I do think the `description` on EPA should be changed (but let me know if you disagree)

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
> Hm, the build failure on Generoso suggests that maybe `Bison` and `Flex` are (build)dependencies too?

hm, this is clearly not ready: it's building the deps, which i intended to avoid.
will mark as draft for now

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot: please test @ generoso EB_ARGS="libpll-2-20230511-GCC-11.3.0.eb"

**Pull request closed without merging**